app/models/work_packages/scopes/relatable.rb

```diff
@@ -40,33 +40,114 @@ module Relatable
       # * Ancestor/descendant: relations cannot be drawn between ancestor and descendants. It is important to note though,
       #   that relations can be created between any two work packages within the same tree as long as they are not in a direct
       #   or transitive ancestor/descendant relationship. So a relation between siblings is just as possible as one between
-      #   aunt and nephew.
+      #   aunt and nephew. Additionally, a transitive relationship which leaves the tree for any one hop, is exempt from that rule
+      #   so it is possible to have child --- blocks ---> other --- blocks ---> parent.
       # * No circles between trees: The ancestor/descendant chain is considered bidirectional when calculating relatability.
       #   This means that starting from the work package both the descendants as well as the ancestors are considered. That
       #   way, relations like
-      #     Child 1 --- parent ---> Parent 1 ---> follows ---> Parent 2 ---> child ---> Child 2 --- follows ---> Child 1
+      #     Child tree 1
+      #       --- parent ---> Parent tree 1
+      #       --- follows ---> Parent tree 2
+      #       --- child ---> Child tree 2
+      #       --- follows ---> Child tree 1
```

Comment:
Following your notation, it would be:

```
Parent --> Parent
   |         |
   v         v
 Child <-- Child

--> is a follows/precedes relation
```